tests/model/test_validate.py

```diff
@@ -0,0 +1,144 @@
+    "parallel-hashmap",  # build.noarch: true (should be generic) but also probably broken on Windows
+    "airflow",  # "grayskull-update" should be "update-grayskull" in conda-forge.yml
+    "ipython_memory_usage",  # bot.inspect should be bot.inspection in conda-forge.yml
+    "rich-argparse",  # grayskull-update should be update-grayskull in conda-forge.yml
+    "htbuilder",  # bot.inspect should be bot.inspection in conda-forge.yml
+    "stats_arrays",  # "grayskull-update" should be "update-grayskull" in conda-forge.yml
+    "textual-fastdatatable",  # bot.inspect should be bot.inspection in conda-forge.yml
+    "aiohttp",  # bot.inspect should be bot.inspection in conda-forge.yml
+    "buildbot",  # bot.inspect should be bot.inspection in conda-forge.yml
+    "sqlalchemy-drill",  # "grayskull-update" should be "update-grayskull" in conda-forge.yml
+    "sphinx-sitemap",  # typo in bot.inspection (conda-forge.yml)
+    "alibabacloud-openapi-util",  # "grayskull-update" should be "update-grayskull" in conda-forge.yml
+    "st-annotated-text",  # bot.inspect should be bot.inspection in conda-forge.yml
+    "buildbot-www",  # bot.inspect should be bot.inspection in conda-forge.yml
+    "wgpu-native",  # bot.abi_migration_branches should be string, not float (conda-forge.yml)
+    "google-ads",  # "grayskull-update" should be "update-grayskull" in conda-forge.yml
+    "dnspython",  # "grayskull-update" should be "update-grayskull" in conda-forge.yml
+    "pyobjc-framework-corebluetooth",  # bot.inspect should be bot.inspection in conda-forge.yml
+    "azure-storage-queue",  # bot.inspect should be bot.inspection in conda-forge.yml
+    "pymc-marketing",  # typo in bot.inspection (conda-forge.yml)
+    "graphite2",  # provider.win has invalid value "win".
+    "lbapcommon",  # provider.osx_arm64 has invalid value "osx_64". See issue #64 of the feedstock.
+    "root",  # provider.osx_arm64 has invalid value "osx_64". See issue #238 of the feedstock.
+    "vector-classes",  # provider.osx_arm64 has invalid value "osx_64". See issue #9 of the feedstock.
+    "espaloma",  # typo in `conda-forge.yml`.azure
+    "sparc-x",  # `conda-forge.yml`.channels is unexpected
+    "jupyter_core",  # `conda-forge.yml`.abi_migration_branches is unexpected, should be moved to `conda-forge.yml`.bot
+    "rerun-sdk",  # unrecognized field `conda-forge.yml`.azure.artifact_retention_days
+    "bamnostic",  # unrecognized field `conda-forge.yml`.build
+    "r-v8",  # unrecognized field `conda-forge.yml`.github.win
+    "python-utils",  # unrecognized field `conda-forge.yml`.dependencies
+    "pyrosm",  # unrecognized option `conda-forge.yml`.build, legacy field `conda-forge.yml`.matrix does not validate
+    "sketchnu",  # `conda-forge.yml`.conda_build.pkg_format may not be None
+    "fd-find",  # invalid `conda-forge.yml`.test_on_native_only value
+    "micromamba",  # invalid value for `conda-forge.yml`.channel_priority
```

Comment:
This field appears allowed: https://github.com/conda-forge/conda-smithy/blob/main/conda_smithy/schema.py#L673

Is this list of known failures out of date?

Reply:
I'm currently updating it, one sec :)

Reply:
should be up to date now @beckermr